meson.build

```diff
@@ -0,0 +1,107 @@
+project(
+	'dunst',
+	'c',
+	version: '1.9.2',
+	license: 'MIT',
+	meson_version: '>=0.60.0',
+	default_options: [
+		'c_std=gnu99',
+		'warning_level=1',
+		'b_ndebug=if-release',
+	],
+)
+
+add_project_arguments([
+	'-pedantic',
+	'-Wno-gnu-zero-variadic-macro-arguments',
+	'-Wno-overlength-strings',
+], language: 'c')
+
+if get_option('buildtype').startswith('debug')
+	add_project_arguments('-DDEBUG_BUILD', language: 'c')
+endif
+
+cc = meson.get_compiler('c')
+
+cairo = dependency('cairo')
+glib = dependency('glib-2.0')
+gio = dependency('gio-2.0')
+gdk_pixbuf = dependency('gdk-pixbuf-2.0')
+pangocairo = dependency('pangocairo')
+x11 = dependency('x11', required: get_option('x11'))
+xinerama = dependency('xinerama', required: get_option('x11'))
+xext = dependency('xext', required: get_option('x11'))
+xrandr = dependency('xrandr', required: get_option('x11'), version: '>=1.5')
+xscrnsaver = dependency('xscrnsaver', required: get_option('x11'))
+systemd = dependency('systemd', required: get_option('systemd'))
```

Comment:
`systemd` is not required to build the project.

Reply:
Yes, it is not required, hence: it requires the `systemd` option to be on, in case the distributor wants to intsall the units directory.

```meson
if systemd.found()
	user_units_dir = systemd.get_variable(pkgconfig: 'systemduserunitdir')
```

Reply:
Why can't installation of the files depend on `get_option('systemd')` then?

Reply:
It does. Again: `dependency` is used so it can retrieve the unit directory.